wordpress-mobile/WordPressKit-iOS
# Add Dangermattic configuration

### Description

This PR adds Dangermattic to this repo.
My main goal was to add RuboCop, but I added some other basic checks as well.
Please let me know if it makes sense to add more checks seen in other repos ([reference](https://github.com/woocommerce/woocommerce-ios/blob/trunk/Dangerfile))

### Testing Details

The main validation is to make sure CI is 🟢.

To run Danger yourself:
1. Checkout this branch 
2. Run `bundle install`
3. Run:
```ruby
DANGER_GITHUB_API_TOKEN=<github_token> bundle exec danger pr <PR_URL>
```
You should get in the console the same errors / warnings reported in the PR.

---
